### DISCUSSION TITLE
Reference Sheet for Concept Art  2

### DISCUSSION THREAD

**Discussion by Swimmablebee912** [gamerevar/proyecto-comunitario-1-unity#10] [Arte]
### Utilizar como referencia también el [Concept Art de Godot](https://github.com/gamerevar/proyecto-comunitario-1-gd/discussions/63)
***
Seguna version de la guia sobre concept art
Mirar tambien la guia de ferchiman:   https://github.com/gamerevar/proyecto-comunitario-1-gd/discussions/63
![ProyectoComunitario_Referencias](https://user-images.githubusercontent.com/76527163/103250593-61642b80-4953-11eb-84d8-87eebb8a1a1d.png)

**Comment by gamerevar** [comment]
# Comentario original de @tatomarinelli 
***
![Technotemple](https://user-images.githubusercontent.com/76547070/103062710-cc95b280-458d-11eb-961c-f0fed1e886b4.png)

Tecno templo concept

https://open.codecks.io/proyecto-comunitario-1-unity/decks/18-arte/card/12z-el-templo
